src/dotnet/CarbonAware.Plugins.NewBasicJsonPlugin/CarbonAwareJsonReaderPlugin.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using CarbonAware.Model;
+using CarbonAware.Plugins.BasicJsonPlugin;
+using Microsoft.Extensions.Logging;
+
+namespace CarbonAware.Plugins.NewBasicJsonPlugin;
+
+public class CarbonAwareJsonReaderPlugin : ICarbonAware
+{
+    private readonly ILogger<CarbonAwareJsonReaderPlugin> Logger;
```

Comment:
We need to follow the same convention with the attributes of the class. Suggested to use `_logger`